be/src/column/field.h

```diff
@@ -176,6 +183,7 @@ class Field {
     int32_t _length = 0;
     uint8_t _short_key_length;
     uint8_t _flags;
+    ColumnUID _uid = -1;
```

Comment:
What's the usage of `_uid`? Is it determined in FE?

Reply:
Use `uid` in DeltaColumnGroup to record which column been updated. This `uid` is came from `TabletColumn`.